src/common/Screen.js

```diff
@@ -137,7 +137,11 @@ class Screen extends PureComponent<Props> {
           contentContainerStyle={[padding && styles.padding]}
         >
           <ScrollView
-            contentContainerStyle={[styles.flexed, centerContent && componentStyles.content, style]}
+            contentContainerStyle={[
+              !scrollEnabled ? styles.flexed : null,
```

Comment:
Let's add a code comment describing the reason for the conditional, just above it. That way the next person won't think it was accidental or poorly considered. 🙂 Maybe something like this:

```diff
           <ScrollView
             contentContainerStyle={[
+              // If `Screen` is responsible for managing scrolling,
+              // keep its childrens' height unbounded. If not, set a
+              // bounded height on its children, e.g., as required for
+              // a child `FlatList` or `SectionList` to work (or even
+              // another `ScrollView`, but hopefully we don't nest too
+              // many of these!).
               !scrollEnabled ? styles.flexed : null,
               centerContent && componentStyles.content,
               style,
```

Reply:
Done.